### DISCUSSION TITLE
Access MAgPIE (data and model)

### DISCUSSION THREAD

**Discussion by Trang1980** [magpiemodel/magpie#393] [General]
Hi everyone, 
I am Trang, a new MAgPIE user. I have not known any thing about MAgPIE, please help me access MAgPIE model. I have some questions:
1. Do we need a licence to access the MAgPIE 4?
2. Could we be able to separate Australia as a country region?
3. I just see Magclass in R, how can I access the Magpie library?
4. If it is possible, please give me guideline to learn about MAgPIE.

Thank you very much for your help.

**Comment by tscheypidi** [comment]
Hi Trang,

1. You don't need a license for MAgPIE itself, but you need a license for GAMS as well as the solver CONOPT4 in order to run the model.
2. This is technically possible, but needs some support from our side as you need to have access to the unaggregated data. I suggest to become first acquainted with the model in standard regions before you shift to another regional aggregation.
3. Have a look at the installation instructions @ https://github.com/magpiemodel/magpie#how-to-install You need to add another package repsository as described there in order to gain access to all relevant packages
4. have a look at our tutorials @ https://magpiemodel.github.io/tutorials/

**Reply by Trang1980** [reply]
Thanks Jan.
